docs/pages/reference/mesheryctl-commands.md

```diff
@@ -171,6 +171,32 @@ Installation, troubleshooting and debugging of Meshery and its adapters.
       </tr>
     {% endfor %}
   {% endfor %}
+  {% for subcommand_hash in site.data.mesheryctlcommands.lifecycle.system.login.command %}{% assign subcommand = subcommand_hash[1] %}

```

Comment:
@warunicorn19 we need to move away from this model of the `mesheryctl` docs repeating themselves over and over and over and over and over and over and over and over.... and over. @alphaX86 was going to pick up on the [branch I had created](https://github.com/meshery/meshery/tree/leecalcote/docs/mesheryctl) to get this effort further down the road. @alphaX86 @adithyaakrishna are either of you going to deliver this or should we reassign?

Reply:
@leecalcote We'll open a new issue regarding YAML restructure as epic if necessary and we'll try to work on it.

@warunicorn19 better make your PR as draft as the YAML will be reworked from today

Reply:
@alphaX86 my PR is in draft itself😂.

Reply:
Yo dude. It's open 😂😂

Reply:
Oh wait does the label `pr/draft` not mean that my PR is in draft?😅

Reply:
Nope. Mark it as draft there will be an option on right sidebar

Reply:
Oh wow, I had no idea😂